### PR TITLE
Add clientset for Output CRD

### PR DESCRIPTION
**Pull request by mugdha-adhav**
## Description
- Added client code for Output CRD using code-generator
- Clientset is generated for non-namespace scoped resource
- `client` packge is auto-generated using code-generator
## Purpose
- The newly added clientset will enable us to create custom-resources of type Output
## Testing Done
- Manually tested for kubernetes 1.15.2
- Made sure that the custom resources of type **Output** are created using the clientset

### CODE REVIEW

**Comment by texascloud** [comment]
What does this change do?

**Comment by smanpathak** [comment]
> What does this change do?

@CamelCaseNotation I am going to add a template for pull request. Devs forget such things when posting reviews!

File: pkg/client/clientset/versioned/clientset.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*
+Copyright The Kubernetes Authors.
```
Comment (smanpathak) [inline pkg/client/clientset/versioned/clientset.go]: Lets change the copyright Platform9 Systems

File: pkg/client/clientset/versioned/clientset.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+package versioned
+
+import (
+	loggingv1alpha1 "github.com/platform9/fluentd-operator/pkg/client/clientset/versioned/typed/logging/v1alpha1"
+	discovery "k8s.io/client-go/discovery"
+	rest "k8s.io/client-go/rest"
+	flowcontrol "k8s.io/client-go/util/flowcontrol"
+)
+
+type Interface interface {
```
Comment (smanpathak) [inline pkg/client/clientset/versioned/clientset.go]: Put comment describing the interface. Also, don't call this "interface"

File: pkg/client/clientset/versioned/clientset.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	*discovery.DiscoveryClient
+	loggingV1alpha1 *loggingv1alpha1.LoggingV1alpha1Client
+}
+
+// LoggingV1alpha1 retrieves the LoggingV1alpha1Client
+func (c *Clientset) LoggingV1alpha1() loggingv1alpha1.LoggingV1alpha1Interface {
+	return c.loggingV1alpha1
+}
+
+// Deprecated: Logging retrieves the default version of LoggingClient.
```
Comment (smanpathak) [inline pkg/client/clientset/versioned/clientset.go]: why is this depreciated? It's strange to see such language in new code we are adding.

File: pkg/client/clientset/versioned/doc.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
+/*
+Copyright The Kubernetes Authors.
```
Comment (smanpathak) [inline pkg/client/clientset/versioned/doc.go]: change to Platform9. See other files for reference.

**Review by smanpathak — CHANGES_REQUESTED** [review]

**Comment by smanpathak** [comment]
@mugdha-adhav Overall, please add unit tests.

File: pkg/client/clientset/versioned/clientset.go (hunk 1 above)
Comment (mugdha-adhav) [reply pkg/client/clientset/versioned/clientset.go]: Made the requested changes

File: pkg/client/clientset/versioned/doc.go (hunk 4 above)
Comment (mugdha-adhav) [reply pkg/client/clientset/versioned/doc.go]: Made the requested changes

File: pkg/client/clientset/versioned/clientset.go (hunk 3 above)
Comment (mugdha-adhav) [reply pkg/client/clientset/versioned/clientset.go]: Removed deprecated implementations. The implementations are anyways not needed for us.

File: pkg/client/clientset/versioned/clientset.go (hunk 2 above)
Comment (mugdha-adhav) [reply pkg/client/clientset/versioned/clientset.go]: Renamed Interface to **LoggingV1alpha1Interface** and added comment

**Comment by mugdha-adhav** [comment]
> What does this change do?

@CamelCaseNotation @smanpathak added the description for the PR

**Review by smanpathak — APPROVED** [review]

**Review by smanpathak — APPROVED** [review]
Ok to submit, after fixing the failing unit test

**Review by smanpathak — APPROVED** [review]

**Review by smanpathak — APPROVED** [review]

**Pull request merged**